src/server.py

```diff
@@ -556,6 +559,46 @@ def run_dailyjob_query():
     return {'dailyjobs': output}
 
 
+# It's a temp endpoint to make database migration from alermanager server to heroku more easier
+@app.route("/data/dump/")
+@json_response
+def run_old_job_dump():
+    limit = request.args.get('limit')
+    offset = request.args.get('offset')
+    date_format = "%04d-%02d-%02d %02d:%02d:%02d"
+    db = create_db_connnection()
+    cursor = db.cursor()
+    query = "select slave, result, build_system_type, duration, platform, buildtype, " \
+            "testtype, bugid, branch, revision, date, failure_classification, failures " \
+            "from testjobs LIMIT %s OFFSET %s" % (limit, offset)
```

Comment:
ok, this is good- 500 seems like a good safe route.  should we add an orderby to help ensure we get accurate data?

on the insert side, how do we ensure we don't have duplicates- I will merge this into alertmanager server, but I need to ensure releng isn't reconfig or anything like that.